HW6_Starter_Code/schema.sql

```diff
@@ -1,2 +1,26 @@
 -- Insert code to create Database Schema
 -- This will create your .db database file for use
```

Comment:
Relationships have not been established in the schema.sql through use of foreign key